server/oauth.ts

```diff
@@ -24,6 +27,13 @@ export const oauthCookieOptions: CookieOptions = {
 	httpOnly: true,
 };
 
+// Zod schema for the claims we expect to find in the ID token
+const IdTokenClaims = z.object({
+	legacy_identity_id: z.optional(z.string()),
+	name: z.optional(z.string()),
+	email: z.optional(z.string()),
+});
```

Comment:
this zod schema would pass for any given object/json passed into this as all the types are optional